src/guide/migration/custom-directives.md

```diff
@@ -57,7 +57,7 @@ const MyDirective = {
   beforeUpdate() {}, // new
   updated() {},
   beforeUnmount() {}, // new
-  unmounted() {}
+  unmounted() {},
```

Comment:
Could you please revert comma changes? We are using Prettier with no-trailing-comma settings 🙏🏻

Reply:
@NataliaTepluhina Oops, sorry, fixed.